app/services/register_and_partner_api/sync_users.rb

```diff
@@ -15,11 +15,15 @@ class SyncUsers
       none: "none",
     }.freeze
 
-    def self.perform
+    def self.perform(all: false)
       new_sync_time = Time.zone.now
       # TODO: Add pagination
       begin
-        sync_users(RegisterAndPartnerApi::User.where(filter: { updated_since: SyncUsersTimer.last_sync }))
+        if all
+          sync_users(RegisterAndPartnerApi::User.all)
+        else
+          sync_users(RegisterAndPartnerApi::User.where(filter: { updated_since: SyncUsersTimer.last_sync }))
+        end
```

Comment:
A ternary? Might be a bit long if it's on one line. Not to fussed either way, i just tend to like ternary's on a single condition

Reply:
I prefer to use ternaries when I deal with expressions. So say
```
title = use_full? ? full_title : short_title
```
But dealing with anything that does side effects (like our `sync_users` method) is a bit scary - cause ternary expression tends to make it harder to focus on which side effects are happening when.

That said, I have an idea.